packages/create-mud/package.json

```diff
@@ -6,24 +6,24 @@
   "license": "MIT",
   "bin": "dist/cli.js",
   "files": [
-    "dist",
-    "templates"
+    "dist"
   ],
   "scripts": {
     "prepare": "pnpm build",
-    "build": "tsup src/cli.ts --minify",
-    "clean": "shx rm -rf lib",
+    "build": "tsup src/cli.ts --minify && ./scripts/copy-templates.sh",
+    "clean": "rimraf dist",
     "dev": "tsup src/cli.ts --watch",
     "prepublishOnly": "npm run clean && npm run build",
-    "test": "echo \"TODO\"",
+    "test": "pnpm test:minimal && pnpm test:react",
+    "test:minimal": "(dist/cli.js test-project --template minimal || true) && rimraf test-project",
+    "test:react": "(dist/cli.js test-project --template react || true) && rimraf test-project",
```

Comment:
would this let the CI fail if creating the template fails or would it still pass bc of the `|| true`?

Reply:
good catch!